src/frontend-proxy.ts

```diff
@@ -61,10 +65,12 @@ export async function frontendProxy(
     (global.REDIRECT_AUTHENTICATED_USERS_TO_LEGACY_BACKEND === 'true' &&
       getCookieValue('authenticated', cookies) === '1')
   )
-    return await fetchBackend(false)
+    return await fetchBackend(false, lang)
```

Comment:
Object Parameter would be more readable:
```suggestion
    return await fetchBackend({ useFrontend: false, lang })
```